test/rest-adapter-custom.test.js

```diff
@@ -8,34 +8,33 @@ var should = require('should');
 
 var DataSource = require('loopback-datasource-juggler').DataSource;
 
-const TEST_ADDRESS = /Bedford Ave.*, Brooklyn, NY 11211, USA/;
-
-describe('REST connector', function () {
-  describe('custom operations', function () {
+var TEST_ADDRESS = /Bedford Ave.*, Brooklyn, NY 11211, USA/;
```

Comment:
const? I know `ES2015` supports it... Anyway Eslint complains about this `const`